games/devtest/mods/unittests/player.lua

```diff
@@ -68,3 +68,19 @@ local function run_player_meta_tests(player)
 	assert(meta:equals(meta2))
 end
 unittests.register("test_player_meta", run_player_meta_tests, {player=true})
+
+--
+-- Player add pos
+--
+local function run_player_add_pos(player)
```

Comment:
```suggestion
local function run_player_add_pos_tests(player)
```

to match the naming scheme of other functions in this file

Reply:
Fixed.